.changeset/strong-toes-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/ea-bootstrap': patch
+---
+
+Applied metrics env var risk mitigations
```

Comment:
Since we are removing an env var, I think we need to bump higher than just `patch`? We should at least clearly call out that this env var is removed, and what impact it has for anyone that used it.

Reply:
Could also be useful to mention here that we've added additional logging for various metrics things